app/javascript/src/patternflyStyles/pf4BaseIE11.scss

```diff
@@ -408,3 +411,26 @@ button:-moz-focusring,
     -ms-grid-column: 2;
   }
 }
+
+.pf-c-nav.pf-m-dark .pf-c-divider {
+  background-color: #4f5255; 
+}
+
+.pf-c-nav.pf-m-dark .pf-c-nav__item.pf-m-current .pf-c-divider {
+    background-color: #212427; }
```

Comment:
Just this line is missing the format now

Reply:
```suggestion
    background-color: #212427;
  }
```

Reply:
@dlabaj please fix this line and we are ready to go.
Don't worry about the docker build issue.

Reply:
I'm merging, I'll fix this once in our repo